spring-jdbc/src/main/java/org/springframework/jdbc/support/KeyHolder.java

```diff
@@ -16,21 +16,21 @@
 
 package org.springframework.jdbc.support;
 
-import java.util.List;
-import java.util.Map;
-
 import org.springframework.dao.InvalidDataAccessApiUsageException;
 import org.springframework.lang.Nullable;
 
+import java.util.List;
+import java.util.Map;
+
 /**
  * Interface for retrieving keys, typically used for auto-generated keys
  * as potentially returned by JDBC insert statements.
  *
- * <p>Implementations of this interface can hold any number of keys.
+ * Implementations of this interface can hold any number of keys.
```

Comment:
Please revert this change.

The `<p>` is there to begin a new paragraph in the generated Javadoc.